app/api/cf-event/services/cf-event.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const axios = require("axios");
+const apiUrl = process.env.CF_CRM_API_URL || 'https://crm.tutor.berlin/api';
```

Comment:
Use strapi to fetch configuration values.
https://docs.strapi.io/developer-docs/latest/setup-deployment-guides/configurations.html